src/stylesheets/elements/form-controls/_date-picker.scss

```diff
@@ -127,18 +196,35 @@ $date-picker-max-width: "mobile";
 
```

Comment:
The icons are displaying larger than expected in HCM. You could move the sizing property on line `118` to `198` like so:

```scss
.usa-date-picker__calendar__previous-year,
.usa-date-picker__calendar__previous-month,
.usa-date-picker__calendar__next-year,
.usa-date-picker__calendar__next-month {
  @extend %usa-date-picker__button;
  background-position: center;
  background-size: auto units(3);
  height: units(3);
  padding: 20px 10px;

  @media (forced-colors: active) {
    mask-size: auto units(3) !important;
  }
}
```

And again later on `398` (you'll need to set the icon here too).


```scss
.usa-date-picker__calendar__previous-year-chunk,
.usa-date-picker__calendar__next-year-chunk {
  @extend %usa-date-picker__button;
  background-position: center;
  background-size: auto units(4);
  margin: auto;
  padding: 40px 0;

  @media (forced-colors: active) {
    mask-size: auto units(4) !important;
  }
}
```

**Before**
![image](https://user-images.githubusercontent.com/3385219/159793134-96d050bb-8686-40d9-b77c-9d40e7796d27.png)

**After**
![image](https://user-images.githubusercontent.com/3385219/159793406-73e0a684-6a35-4db6-b668-ad47bf1297c4.png)

Reply:
Fixed in [97b815f](https://github.com/uswds/uswds/pull/4581/commits/97b815fa5df58ddda7d711b080f94cacfdfc0e2b) !